docs/02-app/02-api-reference/04-functions/cacheLife.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: cacheLife
+description: Learn how to use the cacheLife function to set the cache expiration time for a cached function or component.
+version: canary
+related:
+  title: Related
+  description: View related API references.
+  links:
+    - app/api-reference/next-config-js/dynamicIO
+    - app/api-reference/directives/use-cache
+    - app/api-reference/functions/revalidateTag
+    - app/api-reference/functions/cacheTag
+---
+
+The `cacheLife` function is used to set tge cache lifetime of a function or component. It should be used alongside the `use cache` directive, and within the scope of the function or component.
+
+## Usage
+
+To use `cacheLife`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` file and import `cacheLife` from `next/cache`.
```

Comment:
Can remove the imort suggestion, as you go on to state this with an example under setting the flag
```suggestion
To use `cacheLife`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` file:
```